tests/test_structures.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+import pytest
+
+from requests.structures import CaseInsensitiveDict, LookupDict
+
+
+class TestCaseInsensitiveDict:
+    
+    @pytest.fixture
+    def dict_instance(self):
+        instance = CaseInsensitiveDict()
+        instance['Accept'] = 'application/json'
+        return instance
+
+    def test_list(self, dict_instance):
+        assert list(dict_instance) == ['Accept']
+
+    def test_getitem(self, dict_instance):
+        assert dict_instance['aCCept'] == 'application/json'
```

Comment:
For this kind of test it's really valuable to have quite a few examples with wildly different cases. Something like:

```python
@pytest.mark.parametrize('key', ['aCCept', 'Accept', 'accept', 'ACCEPT', 'aCcEpT'])
def test_getitem(self, dict_instance, key):
    assert dict_instance[key] == 'application/json'
```
```